metadata-models/src/main/pegasus/com/linkedin/dataset/FieldLineageDetails.pdl

```diff
@@ -0,0 +1,18 @@
+namespace com.linkedin.dataset
+
+import com.linkedin.common.Urn
+
+/**
+ * Details of the parent fields and transformation resulting in new field.
+ */
+record FieldLineageDetails {
+    transform: optional string
+
+    parentFields: array[Urn]
```

Comment:
s/parentFields/upstreamFields/?

Reply:
5. Another field for confidence score? cc: @jjoyce0510

Reply:
Point 2 is unclear: the variable viewLineageMcp in "lineage_emitter_view_job_mcp.py" shows upstream columns; am I missing something?